deploy/layered-sre-authorization/rhoam/config.yaml

```diff
@@ -0,0 +1,6 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  matchLabels:
+      api.openshift.com/addon-rhoam-operator: "true"
+      api.openshift.com/addon-rhoam-operator-internal: "true"
```

Comment:
Is addon-rhoam-operator-internal a thing? I thought there was only one?

Reply:
I'm working to verify that operator-internal does not exist in RHOAM

Reply:
This won't exist for GA. Safe to remove and "matchLabelsApplyMode"